api/ims/ims-api.yaml

```diff
@@ -34,6 +34,45 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Error'
+  /sessions:
+    post:
+      description: User login 
+      operationId: login
+      parameters:
+        - name: Email
+          in: query
```

Comment:
please do not use query parameters for sending passwords, more secure is to send them in the request body.